realgud/common/track.el

```diff
@@ -223,59 +223,65 @@ evaluating (realgud-cmdbuf-info-loc-regexp realgud-cmdbuf-info)"
 	 ;; in frame-num. Otherwise, nil.
 	 (frame-num)
 	 (text-sans-loc)
-	 (bp-loc)
 	 (cmdbuf (or opt-cmdbuf (current-buffer)))
 	 )
     (unless (realgud:track-complain-if-not-in-cmd-buffer cmdbuf t)
       (if (realgud:eval-command-p text)
           (realgud:message-eval-results text))
 
-	(if (not (equal "" text))
-	    (with-current-buffer cmdbuf
-	      (if (realgud-sget 'cmdbuf-info 'divert-output?)
-		  (realgud-track-divert-prompt text cmdbuf to))
-	      ;; FIXME: instead of these fixed filters,
-	      ;; put into a list and iterate over that.
-	      (realgud-track-termination? text)
-	      (setq text-sans-loc (or (realgud-track-loc-remaining text) text))
-	      (realgud-track-bp-enable-disable text-sans-loc
-					       (realgud-cmdbuf-pat "brkpt-enable")
-					       't)
-	      (realgud-track-bp-enable-disable text-sans-loc
-					       (realgud-cmdbuf-pat "brkpt-disable")
-					       nil)
-	      (setq frame-num (realgud-track-selected-frame text))
-	      (if (and frame-num (not loc))
-		  (setq loc (realgud-track-loc-from-selected-frame
-			     text cmd-mark)))
-
-	      (setq bp-loc (realgud-track-bp-loc text-sans-loc cmd-mark cmdbuf))
-	      (if bp-loc
-		  (let ((src-buffer (realgud-loc-goto bp-loc)))
-		    (realgud-cmdbuf-add-srcbuf src-buffer cmdbuf)
-		    (with-current-buffer src-buffer
-		      (realgud-bp-add-info bp-loc)
-		      )))
-	      (if loc
-		  (let ((selected-frame
-			 (or (not frame-num)
-			     (eq frame-num (realgud-cmdbuf-pat "top-frame-num")))))
-		    (realgud-track-loc-action loc cmdbuf (not selected-frame)
-                                              shortkey-on-tracing?)
-		    (realgud-cmdbuf-info-in-debugger?= 't)
-                    (realgud-cmdbuf-mode-line-update))
-                (dolist (bp-loc (realgud-track-bp-delete text-sans-loc cmd-mark cmdbuf))
-                  (let ((src-buffer (realgud-loc-goto bp-loc)))
-                    (realgud-cmdbuf-add-srcbuf src-buffer cmdbuf)
-                    (with-current-buffer src-buffer
-                      (realgud-bp-del-info bp-loc)
-                      ))))
-              )
-          )
+      (if (not (equal "" text))
```

Comment:
its easier to look at just the first commit or https://github.com/realgud/realgud/pull/198/files?w=1